code/modules/jobs/job_types/gimmick.dm

```diff
@@ -91,9 +91,7 @@
 	var/chosen_drugs = pick(possible_drugs)
 	var/obj/item/storage/pill_bottle/I = new chosen_drugs(src)
 	H.equip_to_slot_or_del(I,ITEM_SLOT_BACKPACK)
-	var/datum/martial_art/psychotic_brawling/junkie = new //this fits well, but i'm unsure about it, cuz this martial art is so fucking rng dependent i swear...
-	junkie.teach(H)
-	ADD_TRAIT(H, TRAIT_APPRAISAL, JOB_TRAIT)
+	H.gain_trauma_type(rand(BRAIN_TRAUMA_MILD, BRAIN_TRAUMA_SEVERE), rand(TRAUMA_RESILIENCE_SURGERY, TRAUMA_RESILIENCE_LOBOTOMY))
```

Comment:
🤔
 Severe traumas are often game ending, no one wants to spawn fully paralyzed

Reply:
resolved via https://github.com/BeeStation/BeeStation-Hornet/pull/4445/commits/2c7e3bf16b5d7ab94cbf9130104f2b85e1217210